Optimal_bounding_box/include/CGAL/Optimal_bounding_box/oriented_bounding_box.h

```diff
@@ -88,15 +88,17 @@ void construct_oriented_bounding_box(const PointRange& points,
     zmax = (std::max)(rot_pt.z(), zmax);
   }
 
-  obb_points[0] = traits.construct_point_3_object()(xmin, ymin, zmin);
-  obb_points[1] = traits.construct_point_3_object()(xmax, ymin, zmin);
-  obb_points[2] = traits.construct_point_3_object()(xmax, ymax, zmin);
-  obb_points[3] = traits.construct_point_3_object()(xmin, ymax, zmin);
-
-  obb_points[4] = traits.construct_point_3_object()(xmin, ymax, zmax); // see order in make_hexahedron()...
-  obb_points[5] = traits.construct_point_3_object()(xmin, ymin, zmax);
-  obb_points[6] = traits.construct_point_3_object()(xmax, ymin, zmax);
-  obb_points[7] = traits.construct_point_3_object()(xmax, ymax, zmax);
+  const typename Traits::Construct_point_3 cp = traits.construct_point_3_object();
```

Comment:
I think the `const` is an error, because in the concept `Kernel::ConstructPoint_3` does not enforce that the `operator()` of the function is `const`.